Resources/doc/reference/architecture.rst

```diff
@@ -316,43 +316,79 @@ template files, administration panel title and logo.
 Create child admins
 -------------------
 
-Let us say you have a ``PostAdmin`` and a ``CommentAdmin``. You can optionally declare the ``CommentAdmin``
-to be a child of the ``PostAdmin``. This will create new routes like, for example, ``/post/{id}/comment/list``,
-where the comments will automatically be filtered by post.
+Let us say you have a ``PlaylistAdmin`` and a ``VideoAdmin``. You can optionally declare the ``VideoAdmin``
+to be a child of the ``PlaylistAdmin``. This will create new routes like, for example, ``/playlist/{id}/video/list``,
+where the videos will automatically be filtered by post.
 
-To do this, you first need to call the ``addChild`` method in your PostAdmin service configuration :
+To do this, you first need to call the ``addChild`` method in your ``PlaylistAdmin`` service configuration:
 
 .. configuration-block::
 
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
-        <service id="sonata.news.admin.post" class="Sonata\NewsBundle\Admin\PostAdmin">
+        <service id="sonata.admin.playlist" class="AppBundle\Admin\PlaylistAdmin">
             <!-- ... -->
 
             <call method="addChild">
-                <argument type="service" id="sonata.news.admin.comment" />
+                <argument type="service" id="sonata.admin.video" />
             </call>
         </service>
 
-Then, you have to set the CommentAdmin ``parentAssociationMapping`` attribute to ``post`` :
+Then, you have to set the VideoAdmin ``parentAssociationMapping`` attribute to ``playlist`` :
 
 .. code-block:: php
 
     <?php
-    namespace Sonata\NewsBundle\Admin;
+
+    namespace AppBundle\Admin;
 
     // ...
 
-    class CommentAdmin extends AbstractAdmin
+    class VideoAdmin extends AbstractAdmin
     {
-        protected $parentAssociationMapping = 'post';
+        protected $parentAssociationMapping = 'playlist';
 
         // OR
 
         public function getParentAssociationMapping()
         {
-            return 'post';
+            return 'playlist';
+        }
+    }
+
+To display the ``VideoAdmin`` extend the menu in your ``PlaylistAdmin`` class:
+
+.. code-block:: php
+
+    <?php
+
+    namespace AppBundle\Admin;
+
+    class PlaylistAdmin extends AbstractAdmin
```

Comment:
would be great to have the used namespaces included in the example, AbstractAdmin, MenuItemInterface, AdminInterface